tests/next_tests/ffront_tests/test_execution.py

```diff
@@ -801,16 +804,16 @@ def program_domain(
 
     program_domain(a_IJ_float, lower_i, upper_i, lower_j, upper_j, offset_provider={})
 
-    expected = np.asarray(a_IJ_float)
-    expected[1:9, 4:6] = 2 * 2
-
     assert np.allclose(expected, a_IJ_float)
 
 
 def test_domain_tuple(fieldview_backend):
     a_IJ_float = np_as_located_field(IDim, JDim)(np.ones((size, size), dtype=float64))
     b2d_float = a_IJ_float
```

Comment:
this is still weird

Reply:
Based on the function name, not what is supposed to be tested here probably, but I couldn't judge if it should be forbidden either.